plugins/generator-1.15.2/forge-1.15.2/templates/biome.java.ftl

```diff
@@ -163,7 +163,7 @@ import net.minecraft.block.material.Material;
 		}
 
 		@OnlyIn(Dist.CLIENT) @Override public int getFoliageColor() {
-			return ${data.grassColor.getRGB()};
+			return ${data.foliageColor.getRGB()};

```

Comment:
As you constructed JColor with allowNullColor set to true, wrap this in if statement similar to how other color parameters do already